js/src/wrappers/oai.ts

```diff
@@ -203,23 +202,36 @@ interface APIPromise<T> extends Promise<T> {
   withResponse(): Promise<EnhancedResponse>;
 }
 
+const LEGACY_CACHED_HEADER = "x-cached";
```

Comment:
Re-iterating from https://github.com/braintrustdata/braintrust-sdk/pull/270#discussion_r1649757262, but would it be useful to move these headers to the core library, so that other libraries can use the constant?

Reply:
Yeah can do.

Reply:
I just don't want to directly rely on it via the proxy, because it requires publishing an update to core and explicitly depending on that version. I don't think it's worth the headache to propagate a constant like this.

Reply:
Actually they are literally only referenced in these two places. I think I'll just leave it as-is for now.